etc/checksum.factory

```diff
@@ -208,9 +216,8 @@ d64fb84480620115f461ec9aac0ba6be  ./logserver/getjwt.py
 4431fb42281b93c5aa09e760d0910d0b  ./logserver/jwt.php
 24341cffcec0fe15225324f38f707b27  ./logserver/logging_config.json
 eec3476ee8c91ffa71445e4cb860e61f  ./logserver/web-area/put.php
```

Comment:
I see that the checksum entry for `./plugins/EntryConditionGenerator.py` is removed. Is this fine?

Reply:
yes, the file was there before as untracked. It is in 3.11.x